rules/xcodeproj.bzl

```diff
@@ -252,7 +253,22 @@ def _exclude_swift_incompatible_define(define):
         return token
     return None
 
-def _joined_header_search_paths(hmap_paths):
+def _fw_search_paths_for_target(target_name, all_transitive_targets):
```

Comment:
what's `fw` stands for again... sorry i totally forgot

Reply:
no problem. fw = framework. I'll rename the method.